### ISSUE TITLE
jstat documentation

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9-docs#400]
**Issue or pull request number:**
https://github.com/eclipse/openj9/pull/7240

**Overview:**
This is an initial implementation of  `OpenJ9` Java Virtual Machine (`JVM`) statistics monitoring tool.
It adds the `jstat` utility to `OpenJ9` for compatibility with the reference implementation (`RI`).

**Release target:**
`Eclipse OpenJ9 0.18.0`

**Applies to the following JDK versions:**
8 and later

**Applies to the following platforms:**
All platforms

**For new command line options:**

The command syntax:
```
    jstat [<option>] [<vmid>]

  option:
   -J : supply arguments to the Java VM running jstat
   -h : print this help message
   -options : list the available command options
   -class : Classloading statistics

  <vmid>: Attach API VM ID as shown in jps or other Attach API-based tools
NOTE: this utility may significantly affect the performance of the target VM.
At least one option must be selected.
```
- Purpose: obtain statistics information about a Java process

- Restrictions: 
     * This differs from the RI version of `jsat` due to JVM internal differences;
     * This utility is a start point to match `RI` counterpart behaviours as per user demand.

fyi: @DanHeidinga @pshipton

**Comment by pshipton** [comment]
This is for the 0.18 release, updated above. 

I thought -gc option was removed since not implemented yet.

**Comment by JasonFengJ9** [comment]
> I thought -gc option was removed since not implemented yet.

That's correct, I was on an earlier version of `jstat`, removed the option.

**Comment by pmhayward** [comment]
Suggested doc changes:

- Create a new topic in the "Tools" section (file name: `tool_jstat.md`). Easiest to copy one of the other `tool_*.md` files and amend! A couple of things spotted above:
    - "May" tends to imply permission. In most cases, better to use either "can" (ability) or "might" (possibility). In this case, I think it's "might".
    - There is special formatting for "Notes" and "Restrictions". If you can't find them somewhere else in the docs to copy, just ask!
- Add an entry in `mkdocs.yml` for this new topic; follow existing layout and ensure indentation (with spaces) is maintained.
- Update the 0.18.0 "What's New" topic (`version0.18.md`). Add an entry just above "Add more changes here..." and follow the format that's already there. The entry needs to be short and should link to your new topic `tool_jstat.md` for more information. (BTW, it's OK to to refer to a feature being "new" in this topic, but avoid saying that elsewhere - think maintenance!) Ensure that a link to the new entry is added at the top of `version0.18.md` - see the way other entries are done.

**Comment by doveye** [comment]
This topic also needs a section adding for jstat: https://www.eclipse.org/openj9/docs/tool_migration/  
It's a summary of the tools and how they differ in OpenJ9 from in HotSpot, for migrating users.

**Comment by pmhayward** [comment]
Closed by PR #416
